src/Slider.jsx

```diff
@@ -652,14 +652,14 @@ class Rheostat extends React.Component {
             />
           );
         })}
-        {PitComponent && pitPoints.map((n) => {
+        {PitComponent && pitPoints.map((n, index) => {
           const pos = algorithm.getPosition(n, min, max);
           const pitStyle = orientation === 'vertical'
             ? { top: `${pos}%`, position: 'absolute' }
             : { left: `${pos}%`, position: 'absolute' };
 
           return (
-            <PitComponent key={n} style={pitStyle}>{n}</PitComponent>
+            <PitComponent key={`${n}_PC_${index}`} style={pitStyle}>{n}</PitComponent>
```

Comment:
Ok, I took the wrong direction. My first idea was to add small CSS blocs on top of others to manage the height of the pit points (that leads to not unique pit points...)

So if I have to put only 1 pit point per value, how do you manage the height of the point ? When the component is rendered outside the lib, I don't have enough information to calculate its height... 

Thank you,
Sylvain